tests/openthread/Makefile

```diff
@@ -11,43 +12,47 @@ RIOTBASE ?= $(CURDIR)/../..
 # Comment this out to disable code in RIOT that does safety checking
 # which is not needed in a production environment but helps in the
 # development process:
-CFLAGS += -DDEVELHELP -Wall
+#CFLAGS += -DDEVELHELP -Wall
 
 # Change this to 0 show compiler invocation lines by default:
 QUIET ?= 1
 
 USEPKG += openthread
-USEMODULE += openthread_contrib
-USEMODULE += libmbedcrypto
-USEMODULE += libopenthread
-USEMODULE += libopenthread-cli
-USEMODULE += xtimer
+OPENTHREAD_TYPE ?= ftd
+ifeq ($(OPENTHREAD_TYPE),mtd)
+    # MTD: A Minimal Thread Device does not have router functionality
+    # compiled in. As a result, it is not necessary to configure the
+    # routerrole on an MTD. At the same time, an MTD may or may not be sleepy.
+    USEMODULE += openthread-mtd
+    USEMODULE += openthread-cli-mtd
+else
+    # ftd: A Full Thread Device has router functionality compiled in
+    USEMODULE += openthread-ftd
+    USEMODULE += openthread-cli-ftd
+endif
+
+#Define PANID and CHANNEL used by default
+#CFLAGS += -DOPENTHREAD_PANID=0xbeef -DOPENTHREAD_CHANNEL=11
 
 ifneq (,$(filter samr21-xpro,$(BOARD)))
   DRIVER := at86rf233
 endif
-ifneq (,$(filter iotlab-m3 fox iotlab-a8-m3,$(BOARD)))
+ifneq (,$(filter iotlab-m3 fox,$(BOARD)))
```

Comment:
why removing `iotlab-a8-m3` ? This 'board' is very similar to iotlab-m3 and should be kept.